voxel51/apps/api.py

```diff
@@ -228,6 +228,52 @@ def delete_analytic(self, analytic_id):
         res = self._requests.delete(endpoint, headers=self._header)
         _validate_response(res)
 
+    # DATA ####################################################################
+
+    def query_application_data(self, data_query):
+        '''Performs a customized data query at the application level (e.g.
```

Comment:
This should be just one line per https://github.com/google/styleguide/blob/gh-pages/pyguide.md#381-docstrings so I'd move the explanation to a new paragraph.

Reply:
@lethosor thanks for reading the style guide closely!!!!